cefi/handler/capitalcom.py

```diff
@@ -49,19 +49,8 @@ def __init__(
         """
         super().__init__(**kwargs)
         try:
-            if self.testmode:
-                self.client = DemoClient(
-                    log=self.user_id,
-                    pas=self.password,
-                    api_key=self.api_key,
-                )
-            else:
-                self.client = Client(
-                    log=self.user_id,
-                    pas=self.password,
-                    api_key=self.api_key,
-                )
 
+            self._build_client()
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (docstrings):** Please update the docstring for function: `get_bid`

Reason for update: Client is rebuilt on each call to ensure session freshness.

Suggested new docstring:
```
Fetches the bid for the specified instrument. Ensures client session is fresh by rebuilding client.
```